node/subsystem-types/src/messages.rs

```diff
@@ -963,6 +963,9 @@ pub struct HypotheticalFrontierRequest {
 	pub candidates: Vec<HypotheticalCandidate>,
 	/// Either a specific fragment tree to check, otherwise all.
 	pub fragment_tree_relay_parent: Option<Hash>,
+	/// Only return membership if all candidates in the path from the
+	/// root are backed.
+	pub backed_in_path_only: bool,
```

Comment:
@rphmeier I'm out of context with statement distribution, should it be a default option?

Reply:
What do you mean by a default option?

Reply:
Do we only want to set it to `true` when checking incoming collator's advertisements, or is it always the case?